<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLEditMillWidget</class>
 <widget class="QDialog" name="WLEditMillWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit WLMill</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_11">
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::SaveAll</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="1" column="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Smooth</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Angle</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="sbSmoothAng">
            <property name="maximum">
             <double>30.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Search order</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLineEdit" name="leStrFindDrivePos"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pbVerError">
         <property name="text">
          <string>verify error</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Feed</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="sbFbls">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string>mm/min</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>60000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Backlash</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Pause</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="sbHPause">
            <property name="toolTip">
             <string>pause height</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbHPause">
            <property name="text">
             <string>G53 Z</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>speed Pause</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sbPerFpause">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string notr="true">%</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1" rowspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Probe</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QComboBox" name="cbActSafeProbe"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>safe probe</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="WLEditIOWidget" name="editInProbe" native="true"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="cbAutoSetSafeProbe">
            <property name="text">
             <string>autoset SafeProbe</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" rowspan="3">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Input control</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="1" column="0">
           <widget class="WLEditIOWidget" name="editInStop" native="true"/>
          </item>
          <item row="0" column="0">
           <widget class="WLEditIOWidget" name="editInPause" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Input enable</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="WLEditIOWidget" name="editInSD" native="true"/>
          </item>
          <item row="0" column="0">
           <widget class="WLEditIOWidget" name="editInEMG" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Percent manual</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lePercentManual"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>Other</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="4" column="0">
           <widget class="QCheckBox" name="cbUseDriveA">
            <property name="text">
             <string>use Drive A</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbUseGModel">
            <property name="text">
             <string>use GModel</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Heart connect</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Timeout connect</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="cbUseDriveB">
            <property name="text">
             <string>use Drive B</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbAutoStart">
            <property name="text">
             <string>autostart GCode</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSpinBox" name="sbHeartMs">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QSpinBox" name="sbTimeoutMs">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="minimum">
             <number>10000</number>
            </property>
            <property name="maximum">
             <number>30000</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="cbUseMPG">
            <property name="text">
             <string>use MPG</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>Start at...</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_18">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="sbDistZG1">
            <property name="prefix">
             <string>Dist (Z down G1) =</string>
            </property>
            <property name="suffix">
             <string>mm</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="sbFeedZG1">
            <property name="prefix">
             <string>F (Z down G1) = </string>
            </property>
            <property name="suffix">
             <string>mm/m</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>5000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WLEditIOWidget</class>
   <extends>QWidget</extends>
   <header>wleditiowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WLEditMillWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WLEditMillWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbHPause</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbHPause</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
